src/Myriad.Plugins/DUCasesGenerator.fs

```diff
@@ -164,7 +164,8 @@ module internal CreateDUModule =
         ]
 
 
-    let createDuModule (namespaceId: LongIdent) (typeDefn: SynTypeDefn) =
+    let createDuModule (namespaceId: LongIdent) (typeDefn: SynTypeDefn) (config: (string * obj) seq) =
+        printfn "CONFIG: %A" config
```

Comment:
let's get fsliblog in here ;P

Reply:
Lol, all `printfn` I've added is just for debugging, we should remove them if this will be merged